docker/mainnet/bitcoin-cash/bitcoin-cash.mainnet.conf

```diff
@@ -0,0 +1,15 @@
+port=9333
+
+rpcbind=0.0.0.0
+rpcport=9332
+
+rpcallowip=172.100.2.100
+
+txindex=1
```

Comment:
Why do we need txindex in dash?

Reply:
Not sure, but it was in testnet and simnet in previous installations